tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -143,6 +143,14 @@ def kill_all(self):
         self.marionette.switch_to_frame()
         self.marionette.execute_async_script("GaiaApps.killAll();")
 
+    def install_app(self, nameApp, urlApp):
```

Comment:
You don't have the same `install_app` than in [bug 1184415](https://github.com/mozilla-b2g/gaia/pull/31447/files#diff-6d544bbb7c759ffb68f344f348f7740eR146). The best one is in bug 1184415. Here you don't use nameApp.

Then, I'd remove these lines from your PR. Otherwise, you'll get some conflicts.